waspc/src/Wasp/Generator/SdkGenerator/Common.hs

```diff
@@ -46,7 +47,10 @@ mkTmplFd :: Path' (Rel SdkTemplatesDir) File' -> FileDraft
 mkTmplFd path = mkTmplFdWithDst path (SP.castRel path)
 
 sdkRootDirInProjectRootDir :: Path' (Rel ProjectRootDir) (Dir SdkRootDir)
-sdkRootDirInProjectRootDir = [reldir|sdk/wasp|]
+sdkRootDirInProjectRootDir =
```

Comment:
This seems very peculiar, could you give more context here?

Reply:
Added a comment above the function.

Reply:
Understood 👍